MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -130,29 +124,6 @@ void Setup()
 			gd.DepthStencilState = _depthStencilState;
 			gd.RasterizerState = _rasterizerState;
 			gd.SamplerStates[0] = _samplerState;
-			
-			var vp = gd.Viewport;
-            if ((vp.Width != _lastViewport.Width) || (vp.Height != _lastViewport.Height))
-            {
-                // Normal 3D cameras look into the -z direction (z = 1 is in front of z = 0). The
-                // sprite batch layer depth is the opposite (z = 0 is in front of z = 1).
-                // --> We get the correct matrix with near plane 0 and far plane -1.
-                Matrix.CreateOrthographicOffCenter(0, vp.Width, vp.Height, 0, 0, -1, out _projection);
-
-                // Some platforms require a half pixel offset to match DX.
-                if (NeedsHalfPixelOffset)
-                {
-                    _projection.M41 += -0.5f * _projection.M11;
-                    _projection.M42 += -0.5f * _projection.M22;
-                }
-
-                _lastViewport = vp;
-            }
-
-            if (_matrix.HasValue)
-                _matrixTransform.SetValue(_matrix.GetValueOrDefault() * _projection);
-            else
-                _matrixTransform.SetValue(_projection);
 
             _spritePass.Apply();
```

Comment:
does _spritePass.Apply() call SpriteEffect.OnApply()?
Why don't we call  _spriteEffect.Apply() here?

Reply:
I though there was an Effect.Apply() but there isn't. 
This PR looks ok.

Reply:
Thanks!